src/cephadm/cephadm.py

```diff
@@ -3511,10 +3520,7 @@ def deploy_daemon_units(
         # following generated script basically checks if the container exists
         # before stopping it. Exit code will be success either if it doesn't
         # exist or if it exists and is stopped successfully.
```

Comment:
(nit) move this comment into the new `add_stop_action` func?

Reply:
@mgfritch you are right, moved :+1: